code/controllers/subsystem/objects.dm

```diff
@@ -17,14 +17,12 @@ var/datum/subsystem/objects/SSobj
 
 /datum/subsystem/objects/Initialize(timeofday, zlevel)
 	setupGenetics()
-	for(var/atom/movable/AM in world)
-		if (zlevel && AM.z != zlevel)
+	for(var/atom/A in world)
```

Comment:
well technically you CAN have datums in the world... as a subtype of atom (parent_type fuckery) so it still works anyway.